framework/materials/multi_group_xs/opensn_import.cc

```diff
@@ -514,15 +502,15 @@ MultiGroupXS::Initialize(const std::string& file_name)
 
     catch (const std::runtime_error& err)
     {
-      throw std::runtime_error("Error reading Chi cross section file "
+      throw std::runtime_error("Error reading Chi cross-section file "
                                "\"" +
                                file_name + "\".\n" + "Line number " + std::to_string(line_number) +
                                "\n" + err.what());
     }
 
     catch (const std::logic_error& err)
     {
-      throw std::logic_error("Error reading Chi cross section file "
+      throw std::logic_error("Error reading Chi cross-section file "
```

Comment:
**OpenSn** cross-section file

Reply:
Fixed. Every time I think I've gotten all of these, another crops up!